src/theme.tsx

```diff
@@ -0,0 +1,81 @@
+import { type ThemeConfig, extendTheme } from '@chakra-ui/react';
+
+const config: ThemeConfig = {
+  initialColorMode: 'light',
+  useSystemColorMode: false,
+};
+
+const Container = {
+  baseStyle: {
+    maxW: '80ch',
+  },
+};
+
+export const theme = extendTheme({
+  config,
+  space: {
+    min: '0.2rem',
+    0: '0.5rem',
+    1: '0.625rem',
+    2: '1rem',
+    3: '2rem',
+    4: '3rem',
+    5: '4rem',
+    6: '5rem',
+    7: '7rem',
+    8: '9rem',
+  },
+  radii: {
+    sm: '0.4rem',
+    md: '1rem',
+    lg: '2rem',
+  },
+  colors: {
+    pink00: '#E1CCFF',
+    blue00: '#454BFF',
+    black10: '#101326',
+  },
+  // SemanticTokens is used when you want different colors for 'light' and 'dark' mode.
+  semanticTokens: {
+    colors: {
+      background: {
+        default: '#FFFFFF',
+        _dark: '#101326',
+      },
+      green00: {
+        default: '#00837D',
+        _dark: '#C8FAFF',
+      },
+      green10: {
+        default: '#C8FAFF',
+        _dark: '#00837D',
+      },
+    },
+  },
```

Comment:
Which colours are we actually using?

Reply:
Just reduced this down, I think we'll figure that one out later. https://github.com/veltolini/website/pull/20/commits/9a3fc13501940545be21039fe05a24c7503587f9